src/slack/Module.md

```diff
@@ -0,0 +1,113 @@
+# Module Slack
+
+Ballerina Slack connector allows you to work with slack conversations/channels, users, user groups, files and messages through the Slack Web API.
+It handles OAuth 2.0 authentication.
+
+There are basically 5 clients provided by Ballerina to interact with different API groups in Slack Web API.
+1. **slack:Client** - This client is the top most client in the slack module. This can be used to get the relevant client associated with the operation
```

Comment:
```suggestion
1. **slack:Client** - This client is the top-most client in the Slack module. This can be used to get the relevant client associated with the operation
```